src/newrelic.coffee

```diff
@@ -26,7 +25,14 @@
 #   hubot newrelic users - Returns a list of all account users from New Relic
 #   hubot newrelic users email <filter_string> - Returns a filtered list of account users
 #   hubot newrelic users emails - Returns a list of all user emails
-#   hubot newrelic alerts - Returns a list of active alert violations
+#   hubot newrelic alerts - Returns a list of active alert violations matching the current channel's subscriptions
+#   hubot newrelic alerts all - Returns a list of all active alerts
+#   hubot newrelic alerts subscribe <pattern> subscribe the current channel to alerts matching <pattern>
+#   hubot newrelic alerts unsubscribe <subscription_id> remove an existing subscription
+#   hubot newrelic alerts subscriptions - show the current channel's subscriptions
+#   hubot newrelic alerts set <variable> - enable an optional alert setting, like "verbose"
+#   hubot newrelic alerts unset <variable> - disable an optional alert setting, like "verbose"
+#   hubot newrelic alerts show <variable> - display the value of an optional alert setting, like "verbose"
```

Comment:
`variable` -> `setting` in the three lines above?

Reply:
fixed!

Reply:
oh you know what, I was looking at the code, not the comments. I'm not sure variable is a bad term to use in that context?

Reply:
You know what, I will change that.